### ISSUE TITLE
Building the qt client in Chat demo can't pass  

### ISSUE THREAD

**Issue by tmacychen** [zeroc-ice/ice-demos#29]
qt version 5.6.1

ice version 3.6.3
[demos link](https://github.com/zeroc-ice/ice-demos/tree/3.6/cpp/Chat/qtClient)
problems:

1. `make` for this project directory can't work before i manually run `/usr/bin/slice2cpp -I. -I/usr/share/slice Chat.ice` ,`ChatSession.ice:10: error: Can't open include file "Ice/BuiltinSequences.ice"` and `/usr/bin/slice2cpp -I. -I/usr/share/slice PollingChat.ice`
2. and then`qmake` goes well
3. then `make` comes out some errors like "ChatSession.cpp:394：对‘IceInternal::Cpp11FnCallbackNC::Cpp11FnCallbackNC(std::function<void (IceUtil::Exception const&)> const&, std::function<void (bool)> const&)’未定义的引用(Undefined reference)"

did I miss some packages?

btw, I pass the cmdLineClient building and goes well. Thanks

**Comment by pepone** [comment]
Hi,

> Can't open include file "Ice/BuiltinSequences.ice" and

This indicate that you have not installed Ice in a standard location, you need to set ICE_HOME environment variable to point to your Ice distribution.

**Comment by tmacychen** [comment]
> Hi,
> 
> > Can't open include file "Ice/BuiltinSequences.ice" and
> 
> This indicate that you have not installed Ice in a standard location, you need to set ICE_HOME environment variable to point to your Ice distribution.

Hi,pepone:
thanks to your reply!
I export the ICE_HOME="/usr" for my system, but it doesn't work for me .
the package libzeroc-ice-dev acctully install the file in the  "/usr"
there are something wrong in qt project maybe.Could you building well in ice 3.6?

**Comment by pepone** [comment]
Hi,

I can give it a try, What operating system and version are you using?

**Comment by tmacychen** [comment]
> Hi,
> 
> I can give it a try, What operating system and version are you using?

I use deepin which is a Linux distrobution  in China . And you can use Debian 9 instead.

**Comment by tmacychen** [comment]
> Hi,
> 
> I just tested Debian 9 and the demo build without problems, what Ice package version are you using? you can check with `dpkg-query -l | grep zeroc` it should look something like:
> 
> ```
> dpkg-query -l | grep zeroc
> ii  libzeroc-freeze3.6:amd64         3.6.3-5                        amd64        C++ run-time for the Freeze persistence service
> ii  libzeroc-ice-dev:amd64           3.6.3-5                        amd64        libraries and headers for developing Ice applications in C++
> ii  libzeroc-ice3.6:amd64            3.6.3-5                        amd64        C++ run-time libraries for the Ice framework
> ii  zeroc-ice-compilers              3.6.3-5                        amd64        Slice compilers for developing Ice applications
> ii  zeroc-ice-slice                  3.6.3-5                        all          Slic

**Comment by pepone** [comment]
> Did you use the chat Qt demo?
Yes I did, can you start with a clean demo and paste the full output of the `make` command

```
cd cpp/Chat/qtClient
git clean -dxf
make
```
